### PR TITLE
Pc admin take2

### PR DESCRIPTION
**Pull request by pconrad**
In this PR, we add the ability to designate admins through an environment variable:

```
app.admin.email=phtcon@ucsb.edu,bkb@ucsb.edu
```

The idea is not necessarily that this is the *only* way to create admins; rather it is a way to bootstrap an initial set of admins that can then (eventually, if/when this feature is added), designate other users as admins.

### CODE REVIEW

File: src/main/java/edu/ucsb/cs156/kitchensink/controllers/UsersController.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+public class UsersController extends ApiController {
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    ObjectMapper mapper;
+
+    @ApiOperation(value = "Get a list of all users")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @GetMapping("/")
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/kitchensink/controllers/UsersController.java]: Try to be consistent between this and the `@GetMapping("")` in `src/main/java/edu/ucsb/cs156/kitchensink/controllers/UserInfoController.java`

**Review by sethvanb — APPROVED** [review]
LGTM

File: frontend/src/main/layouts/BasicLayout/Navbar.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ export default function Navbar() {
   const { data: currentUser } = useCurrentUser();
   const doLogout = useLogout();
 
+  console.log('currentUser',currentUser);
```
Comment (kphamm) [inline frontend/src/main/layouts/BasicLayout/Navbar.js]: do you want to keep this console.log?

**Review by kphamm — APPROVED** [review]
I made one comment but otherwise LGTM

File: frontend/src/main/layouts/BasicLayout/Navbar.js (hunk 2 above)
Comment (pconrad) [reply frontend/src/main/layouts/BasicLayout/Navbar.js]: Good catch.. I've removed it.

**Pull request merged**